lib/plugins/chat.js

```diff
@@ -1,8 +1,59 @@
 module.exports.player = function(player, serv, settings) {
   player._client.on('message', function(packet) {
-    serv._writeAll('message', {
+    if(packet.message.split('')[0] == "/") {
```

Comment:
What's the point of splitting the string? Couldn't it just be `packet.message[0]`?